### PR TITLE
enh(doc): Improve Centreon IMP chapter

### PR DESCRIPTION
**Pull request by lpinsivy**
* Update images
* Correct typo on FR chapter
* Improve difference between pack and probes

### CODE REVIEW

File: doc/en/configuration_guide/imp/index.rst [hunk 1]
```diff
@@ -184,7 +185,11 @@ Don't forget that each pack use a plugin to work. Each monitoring procedure will
 install the plugin needed.
 
 .. note::
-    Install plugins on each poller required. Otherwise your supervision will not work.
+    Install plugins on each poller required. Otherwise your supervision will
```
Comment (sc979) [inline doc/en/configuration_guide/imp/index.rst]: Install plugins on each poller required -> Installing plugins on each poller is required

File: doc/en/configuration_guide/imp/index.rst [hunk 2]
```diff
@@ -12,9 +12,10 @@ working condition your Centreon platform. Templates of monitoring ready to use i
 procedure that allow to start your monitoring less than 30 minutes after the installation of your
 Centreon platform.
 
-Centreon Plugins' project offer a large range of functionality and is one of the richest in the
-market: +170 environments is around 2000 monitoring indicators which allow to deploy your monitoring
-easily and quickly.
+The Plugin Packs (configuration pack), developed by Centreon, are based on the
+Centreon Plugins (monitoring probes), offer a large range of functionality and
+is one of the richest in the market: +170 environments is around 2000 monitoring
```
Comment (sc979) [inline doc/en/configuration_guide/imp/index.rst]: +170 environments is around 2000 monitoring -> +170 environments for almost 2000 monitoring

File: doc/fr/configuration_guide/imp/index.rst [hunk 3]
```diff
@@ -140,7 +141,7 @@ sur "Try it" dans la première colonne du tableau comparatif des offres.
     :align: center
 
 Créez maintenant votre compte Centreon IMP (rubrique New customer) et conservez
-précieusement vos login et password. Cela vous est nécessaire pour activer votre
+précieusement votre login et password. Cela vous est nécessaire pour activer votre
```
Comment (sc979) [inline doc/fr/configuration_guide/imp/index.rst]: Cela vous est nécessaire -> Ils vous seront nécessaire

File: doc/fr/configuration_guide/imp/index.rst [hunk 4]
```diff
@@ -228,8 +229,11 @@ avez la liste des dépendances sous forme de rpm à installer pour que votre
 supervision puisse démarrer.
 
 .. note::
-    Pensez à déployer tous les rpms sur chaque poller qui le nécessitent. Sinon votre supervision ne
-    pourra pas fonctionner.
+    Pensez à déployer tous les rpms sur chaque poller qui le nécessitent.
+    Sinon votre supervision ne pourra pas fonctionner. En effet, seuls les
+    Centreon Plugins des 11 premiers Plugin Packs ont été installés par défaut
```
Comment (sc979) [inline doc/fr/configuration_guide/imp/index.rst]: Centreon Plugins -> Plugins Centreon

File: doc/en/configuration_guide/imp/index.rst (hunk 1 above)
Comment (lpinsivy) [reply doc/en/configuration_guide/imp/index.rst]: corrected

File: doc/en/configuration_guide/imp/index.rst (hunk 2 above)
Comment (lpinsivy) [reply doc/en/configuration_guide/imp/index.rst]: corrected

File: doc/fr/configuration_guide/imp/index.rst (hunk 3 above)
Comment (lpinsivy) [reply doc/fr/configuration_guide/imp/index.rst]: corrected

File: doc/fr/configuration_guide/imp/index.rst (hunk 4 above)
Comment (lpinsivy) [reply doc/fr/configuration_guide/imp/index.rst]: corrected

**Comment by lpinsivy** [comment]
@sc979 changes applied

**Pull request merged**